### DISCUSSION TITLE
File Structure

### DISCUSSION THREAD

**Discussion by diehlmitchell** [krjo/shopify-webpack-dev-workflow#21] [General]
unsure of how the file structure is supposed to look during and at the end of the process

**Comment by krjo** [comment]
Hi, can you specify your question? The webpack config builds your files from src/ into a Shopify ready theme structure in dist/.

**Reply by diehlmitchell** [reply]
I have since resolved my issue, however I would like (or would've haha) more specific details on the readme. On another note, once I get to, " _npm start_", I am getting errors, which is shown in the image attached if you would know anything about that
![some stupid error] (https://user-images.githubusercontent.com/73497540/112538871-b4991f00-8d86-11eb-8bb1-9f0d90cc9d87.PNG)

**Reply by krjo** [reply]
I haven't gotten that error before. What version of node are you using? Try updating to 14.16.0, deleting node_modules, running npm install again, then npm start.

**Reply by diehlmitchell** [reply]
unfortunately this fix did not work :/, however I can still get functionality by building, and deploy commands, which do work
